crates/diesel_models/src/schema.rs

```diff
@@ -227,6 +227,8 @@ diesel::table! {
         modified_at -> Timestamp,
         #[max_length = 64]
         address_id -> Nullable<Varchar>,
+        #[max_length = 255]
```

Comment:
Nit, we probably don't want an upper limit of 255 on the length of the id. `64` should serve us well enough.